install_config/configuring_authentication.adoc

```diff
@@ -1053,6 +1053,42 @@ a value is used as the display name. Optional, case-insensitive.
 identity determined from the headers specified in `headers`. The first header containing
 a value is used as the preferred user name when provisioning. Optional, case-insensitive.
 
+[discrete]
+[[windows-sspi-using-request-header]]
+==== SSPI connection support on Windows
+
+ifdef::openshift-enterprise[]
+
+[IMPORTANT]
+====
+Using SSPI connection support on Windows is a Technology Preview feature.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+endif::[]
+
+Starting in version
+ifdef::openshift-enterprise[]
+3.11,
+endif::[]
+ifdef::openshift-origin[]
+1.11,
+endif::[]
+SSPI connection support for request header connections is enabled.
```

Comment:
Starting in version 3.11, `oc` supports the Security Support Provider Interface (SSPI) to allow for SSO flows on Windows. If you use the request header identity provider with a GSSAPI enabled proxy to connect an Active Directory server to OpenShift Container Platform, users can automatically authenticate to OpenShift Container Platform through the `oc` command line interface from a domain-joined Windows computer.